Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -136,7 +136,7 @@
 "wikitext-upload-save-anonymously" = "Button text for publishing without logging in";
 "wikitext-upload-save-anonymously-warning" = "Button sub-text informing user or draw-backs of not signing in before saving wikitext. Parameters:\n* $1 - sign in button text";
 "wikitext-upload-save-terms-and-license" = "Button text for information about the Terms of Use and edit license. Parameters:\n* $1 - 'Terms of Use' link ([[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-name]])\n* $2 - license name link";
-"wikitext-upload-save-terms-name" = "This message is used as $1 in the message [[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-and-license]].\n{{Identical|Terms of use}}";
+"wikitext-upload-save-terms-name" = "This message is used in the message [[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-and-license]].\n{{Identical|Terms of use}}";
```

Comment:
Ya that's a tough call... I'm 50/50 about being able to reference other "$" keys in the QQQ, but then, that's the only QQQ value that does this, right, so maybe it's not so valuable...